website/docs/cloud-docs/api-docs/private-registry/gpg-keys.mdx

```diff
@@ -38,7 +38,7 @@ description: >-
 
 # GPG Keys API
 
-These endpoints are only relevant to private providers. When you [publish a private provider](/terraform/cloud-docs/registry/publish-providers) to the HCP Terraform private registry, you must upload the public key of the GPG keypair used to sign the release. Refer to [Preparing and Adding a Signing Key](/terraform/registry/providers/publishing#preparing-and-adding-a-signing-key) for more details.
+These endpoints are only relevant to private providers. When you [publish a private provider](/terraform/cloud-docs/registry/publish-providers) to the HCP Terraform private registry, you must upload the public key of the GPG keypair used to sign the release. Only RSA/DSA formats are supported. Refer to [Preparing and Adding a Signing Key](/terraform/registry/providers/publishing#preparing-and-adding-a-signing-key) for more details.
```

Comment:
Quick suggestion to avoid active voice:
```suggestion
These endpoints are only relevant to private providers. When you [publish a private provider](/terraform/cloud-docs/registry/publish-providers) to the HCP Terraform private registry, you must upload the public key of the GPG key-pair that you used to sign the release. The HCP Terraform registry supports RSA or DSA formatted GPG keys. Refer to [Preparing and adding a signing key](/terraform/registry/providers/publishing#preparing-and-adding-a-signing-key) for more details.
```